### PR TITLE
Added authentication using email test case

### LINKED ISSUE #179: Add test cases
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in our slack channel at https://opensourcehelp.slack.com/. Request invite [here](https://opensourcehelp.herokuapp.com)

**Description**:
- Add test case for authentication using Email
- Add test case for contest form and Chat session form

**Would you like to work on this issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #179

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@tapasweni-pathak How could i include `authentication` in travis?

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ before_script:
   - python manage.py collectstatic --noinput
 
 script:
-  - coverage run --source='main' manage.py test --verbosity 2
+  - coverage run --source=main,authentication  manage.py test --verbosity 2
```
Comment (tapaswenipathak) [inline .travis.yml]: What happens if you say ```coverage run oshc/manage.py test```?

Comment (vaibhavsingh97) [reply .travis.yml]: Then our coverage decreased to 42% because Travis makes virtual environment and `test.py` only covers some file

**Comment by vaibhavsingh97** [comment]
@tapasweni-pathak please review and merge

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]
We won't be going ahead with writing the app names specifically. 
```coverage run --source='.' manage.py test``` runs good on all apps but also on where we don't want it to.

Will merge this with a .coveragerc and including the excludes of not important files/directories; in process [here](https://github.com/tapasweni-pathak/OpenSourceHelpCommunity.github.io).

**Comment by vaibhavsingh97** [comment]
@tapasweni-pathak So i will push `.coveragerc` 👍

**Comment by vaibhavsingh97** [comment]
@tapasweni-pathak Coverage reduced as it's checking the whole code

**Comment by jarifibrahim** [comment]
@vaibhavsingh97 Ping! Are you still working on this?

**Review by tapaswenipathak — APPROVED** [review]

**Pull request merged**